jobs/tendrl-build-triggers.yml

```diff
@@ -92,3 +92,27 @@
           - project: 'tendrl-build-X-build_server-teardown'
             property-file: env.sh
             condition: ALWAYS
+
+
+- project:
+    name: 'release'
+    project_name: 'release'
+    target_repo: 'tendrl-test'
+    branch: 'release/1.4.0'
+    build_commands: 'make dist && make srpm'
+    clustername: build_server
+    copr_config: '~/dahorak_config_copr'
```

Comment:
Hope you can use our official copr-cli token

[copr-cli]
login = pauaulbznyvkjgkpyeja
username = tendrl
token = rjfneixsaxwympcmoukrriyvkemksr
copr_url = https://copr.fedorainfracloud.org
# expiration date: 2017-11-18